tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2016",
+    "target": "ES2018",
```

Comment:
The  change to `tsconfig.json` is to be able to learn Javascript/Typescript along w/ `hardhat.ethers` and `chai` in a Typescript REPL